README.md

```diff
@@ -3,17 +3,7 @@ CHT Pipeline
 
 CHT Pipeline is a tool used to define data models for transforming the raw data we get from Couch DB into models that can then be queried to build dashboards
 
-# Installation
-This project can be run in one of two ways:
-1. Using the docker image and passed as an argument to [cht-sync](https://github.com/medic/cht-sync)
-2. Using the local installation of DBT
+# Dev installation instructions
 
-## Docker
 1. Follow the instructions in [cht-sync](https://github.com/medic/cht-sync) to set up the required environment variables
-1. Add the path to your pipeline branch to the [docker-compose.postgres.yml](https://github.com/medic/cht-sync/blob/main/docker-compose.postgres.yml#L13) file in the cht-sync project
 1. Run `docker-compose up` to run the pipeline
```

Comment:
Let's clarify if I have to run it on the cht-pipeline directory or the cht-sync directory.